docs/package.json

```diff
@@ -106,7 +106,6 @@
     "react-select": "^3.0.4",
     "react-spring": "^8.0.27",
     "react-swipeable-views": "^0.13.3",
-    "react-test-renderer": "^16.8.5",
```

Comment:
I moved this here on accident. It was only ever used in `@material-ui/core/test-utils/describeConformance`.

Reply:
@dependabot squash and merge